plugins/module_utils/backup.py

```diff
@@ -76,29 +78,32 @@ def mvs_file_backup(dsn, bk_dsn=None, tmphlq=None):
             bk_dsn = extract_dsname(dsn) + "({0})".format(temp_member_name())
 
         bk_dsn = _validate_data_set_name(bk_dsn).upper()
-        response = datasets._copy(dsn, bk_dsn)
-        if response.rc != 0:
+        try:
+            rc = datasets.copy(dsn, bk_dsn)
```

Comment:
This variable `rc` which is the ZOU response object is not used anyplace, not sure if linters will complain about an unused variable, otherwise its not really an issue.

Reply:
oops! removed it now, actually linters didn't showed a problem, maybe because is populated again down below, but is assigned value was never used so I removed it.